KyberElasticAdapter/contracts/KyberElasticAdapter.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "./UniswapV3likeAdapter.sol";
+
+interface IKyberPool {
+    function token0() external view returns (address);
+
+    function token1() external view returns (address);
+
+    function swap(
```

Comment:
Missing NatSpec
Add descriptions for the interface, events and functions
Refer https://docs.soliditylang.org/en/v0.8.17/natspec-format.html